packages/now-cli/test/dev/integration.js

```diff
@@ -622,14 +649,6 @@ test(
     });
   })
 );
-/*
```

Comment:
I'm not sure why this was commented out, but removed since it's dead code.

Reply:
It might be a test that is inconsistent between production and development